2012-01-22  Pino Toscano  <toscano.pino@tiscali.it>

	* sysdeps/mach/hurd/socket.c (__socket): Return EAFNOSUPPORT instead
	of the non-standard EPFNOSUPPORT.

---
 sysdeps/mach/hurd/socket.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/sysdeps/mach/hurd/socket.c
+++ b/sysdeps/mach/hurd/socket.c
@@ -57,7 +57,7 @@
      isn't supported.  */
   if (err == MACH_SEND_INVALID_DEST || err == MIG_SERVER_DIED
       || err == MIG_BAD_ID || err == EOPNOTSUPP)
-    err = EPFNOSUPPORT;
+    err = EAFNOSUPPORT;
 
   if (err)
     return __hurd_fail (err);
